src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2820,8 +2820,7 @@ def webapp_up(cmd, name, resource_group_name=None, plan=None, location=None, sku
         site_config = client.web_apps.get_configuration(rg_name, name)
     else:  # need to create new app, check if we need to use default RG or use user entered values
         logger.warning("webapp %s doesn't exist", name)
-        sku_value = get_sku_to_use(src_dir, sku)
-        sku = get_sku_name(sku_value)
+        sku = get_sku_to_use(src_dir, sku)
```

Comment:
Why do you switch from sku name to sku value

Reply:
if you see line 2824 - we initially did this on a separate line but we need this only when we display the SKU - to use SKU name otherwise in the commands we use the SKU value - so made that change